nix/mysql/default.nix

```diff
@@ -216,7 +216,7 @@ in
 '';
 
             configureScript = pkgs.writeShellScriptBin "configure-mysql" ''
-              PATH="${lib.makeBinPath [config.package pkgs.coreutils]}:$PATH"
+              PATH="${lib.makeBinPath [config.package pkgs.coreutils pkgs.findutils]}:$PATH"
```

Comment:
Any reason not to just use `writeShellApplication` and put these in `runtimeInputs`?

Reply:
We probably wanna use it anyway for shellcheck safety.

Reply:
> Any reason not to just use writeShellApplication and put these in runtimeInputs?

Nothing specific, I just kept the changes specific to the PR description.

Reply:
I can change it to `writeShellApplication` it shouldn’t be a problem. 

Or maybe, I can merge this one and create one PR that will change it in all the places.

Reply:
> Or maybe, I can merge this one and create one PR that will change it in all the places.

👍

Reply:
https://github.com/juspay/services-flake/pull/155